Create ui-c8y-1021-49-10-the-alarm-page-of-a-service-lists-all-the-alarms-of.md (#3057)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-49-10-the-alarm-page-of-a-service-lists-all-the-alarms-of.md

---
date: ""

> title: Service alarm page showed alarms from other sources

Reviewer: ```suggestion
title: Alarms tab of a service no longer shows alarms from other sources
```

> title: The alarm page of a service lists all the alarms of the tenant

Reviewer 2: ```suggestion
title: Service alarm page showed alarms from other sources
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4429
version: 1021.49.10
---

> The alarm page of a service lists all the alarms of the tenant

Reviewer 2: ```suggestion
Previously, the alarm page of a service displayed alarms from other sources except the service in context itself. This has now been fixed and only alarms related to the service are displayed.
```

> Previously, the alarm page of a service displayed alarms from other sources except the service in context itself. This has now been fixed and only alarms related to the service are displayed.

Reviewer: ```suggestion
Previously, the **Alarms** tab of a service displayed alarms from other sources than the respective service. This issue has been fixed and now only alarms related to the service are displayed.
```

> Previously, the **Alarms** tab of a service displayed alarms from other sources than the respective service. This issue has been fixed and now only alarms related to the service are displayed.

Reviewer: ```suggestion
Previously, the **Alarms** tab of a service in the device details displayed alarms from other sources than the respective service. This issue has been fixed and now only alarms related to the service are displayed.
```
